src/js/player.js

```diff
@@ -99,7 +99,11 @@ vjs.Player = vjs.Component.extend({
 
     if (options['plugins']) {
       vjs.obj.each(options['plugins'], function(key, val){
-        this[key](val);
+        try {
```

Comment:
rather than using try/catch, we can see whether `this[key]` is a function and then log an error. This way it continues working even if a plugin is missing but we still have a better error log.